bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -26,81 +31,639 @@ Tested TV models:
+
+On legacy TV's, you may see an error like this:
+
+```
+2021-12-08 12:19:50.262 [DEBUG] [port.upnp.internal.UpnpIOServiceImpl] - Error reading SOAP response message. Can't transform message payload: org.jupnp.model.action.ActionException: The argument value is invalid. Invalid number of input or output arguments in XML message, expected 2 but found 1.
+```
+
+This is not an actual error, but is what is returned when a value is polled that does not yet exist, such as the URL for the TV browser, when the browser isn’t running. These messages are not new, and can be ignored. Enabling `subscription` will eliminate them.  
+
+Some channels do not work on some TV's. It depends on the age of your TV, and what kind of interface it has. Only link channels that work on your TV, polling channels that your TV doesn't have may cause errors, and other problems.
+
+If you see errors that say `no route to host` or similar things, it means your TV is off. The binding cannot discover, control or poll a TV that is off.
+
+The `getSupportedChannelNames` messages are not UPnP services, they are not actually services that are supported *by your TV* at all. They are the internal capabilities of whatever method is being used for communication (which could be direct port connection, UPnP or websocket). 
+They also do not reflect the actual capabilities of your TV, just what that method supports, on your TV, they may do nothing.
+
+You should get `volume` and `mute` channels working at the minnimum. Other channels may or may not work, depending on your TV and the binding configuration.
+
+### Separating the Samsung logging into its own file
+
+To separate all the Samsung logging information into a separate file, please edit the file `userdata/etc/log4j2.xml` as follows:
+
+1. Add an logger appender definition (including the log file name)
+2. Add a logger definition referencing the appender defined in step 1
+
+Example for logging all DEBUG logs into a separate file `samsungtv.log` under the standard log folder
+
+```
+<Appenders>
+...
+    <!-- SamsungTV binding appender -->
+    <RollingFile fileName="${sys:openhab.logdir}/samsungtv.log" filePattern="${sys:openhab.logdir}/samsungtv-%d{yyyyMMdd-HHmmss}.log" name="SAMSUNGTV">
+        <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} [%-5.5p] [%-36.36c] - %m%n"/>
+        <Policies>
+            <OnStartupTriggeringPolicy/>
+            <SizeBasedTriggeringPolicy size="16 MB"/>
+        </Policies>
+    </RollingFile>
+</Appenders>
+<Loggers>
+...
+    <Logger additivity="false" level="DEBUG" name="org.openhab.binding.samsungtv">
+        <AppenderRef ref="SAMSUNGTV"/>
+    </Logger>
+</Loggers>
+```
+
+If you have problems with the binding, set the log level to `TRACE` (in place of `DEBUG`) and post a message with a TRACE log covering 30 seconds before the issue and 30 seconds after (please don't send me a log with one line that you think is relevant in it, I can't tell much from this). 
```

Comment:
I think this whole section should be removed. Information on how to setup logging is not binding specific and should be documented, maintained and published at a central place, Only list binding specifics to logging.

Reply:
Removed